core/cautils/kustomizedirectory_test.go

```diff
@@ -12,37 +13,29 @@ func TestGetKustomizeDirectoryName(t *testing.T) {
 	tests := []struct {
 		name                string
 		args                args
-		createKustomization bool // create kustomization.yml file in the path
 		want                string
+		createKustomization bool
 	}{
 		{
-			name: "kustomize directory without trailing slash",
+			name: "kustomize directory",
 			args: args{
-				path: "/tmp",
+				path: os.TempDir(),
```

Comment:
Just a nitpick, but for this purpose you can use testing package TempDir (which is cleaned up when the test is completed) https://pkg.go.dev/testing#T.TempDir